### PR TITLE
chore(files): Switch widget icon to primary color

### PR DESCRIPTION
**Pull request by Pytal**
## Summary

- Switch widget to primary color
image

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by szaimen — APPROVED** [review]
LGTM but didnt test

**Review by jancborchardt — CHANGES_REQUESTED** [review]
Folder icons should indeed be primary color, but text documents not. :) If it’s a .doc/.odt, it has a distinct blue color, not primary (and this seems to be plain text, right?) @Pytal the correct coloring is in https://github.com/nextcloud/server/issues/44340#issue-2195826948

File: apps/files/src/views/ReferenceFileWidget.vue [hunk 1]
```diff
@@ -57,8 +57,8 @@
 		@click="navigate">
 		<span class="widget-file__image" :class="filePreviewClass" :style="filePreviewStyle">
 			<template v-if="!previewUrl">
-				<FolderIcon v-if="isFolder" :size="88" />
-				<FileIcon v-else :size="88" />
+				<FolderIcon v-if="isFolder" :size="88" fill-color="var(--color-primary-element)" />
+				<FileIcon v-else :size="88" fill-color="var(--color-primary-element)" />
```
Comment (susnux) [inline apps/files/src/views/ReferenceFileWidget.vue:61]:
```suggestion
				<FileIcon v-else :size="88" />
```

**Review by jancborchardt — APPROVED** [review]
Thank you! :)

**Review by susnux — APPROVED** [review]

**Pull request merged**